docs/contracts/v4/guides/11-position-manager.mdx

```diff
@@ -0,0 +1,838 @@
+        ""                // No hook data needed
+    );
+```
+
+Set up delta resolution parameters:
+
+```solidity
+if (useFeesAsLiquidity) {
+        // Using CLOSE_CURRENCY for automatic handling of each token
+        params[1] = abi.encode(currency0);  // Handle token0
+        params[2] = abi.encode(currency1);  // Handle token1
+    } else {
+        // Standard SETTLE_PAIR for providing tokens
+        params[1] = abi.encode(currency0, currency1);
+    }
+```
+
+Execute the operation:
+
+```solidity
+// Execute the increase
+    positionManager.modifyLiquidities(
+        abi.encode(actions, params),
+        block.timestamp + 60  // 60 second deadline
+    );
+}
+```
+
+## Decreasing Liquidity
+
+When you want to remove liquidity from a position, you’ll need to handle both the liquidity reduction and any accumulated fees. Let’s understand how to implement this effectively.
+
+### **Understanding the Operation**
+
+Decreasing liquidity involves:
+
+1. Specifying how much liquidity to remove
+2. Setting minimum amounts to receive (slippage protection)
+3. Collecting both the removed liquidity and any accumulated fees
+
+### **Delta Resolution Options**
+
+When decreasing liquidity, you’ll receive tokens, so we have different options for handling the positive deltas:
+
+1. **Standard Case**: Receive all tokens
+
+```solidity
+bytes memory actions = abi.encodePacked(
+    Actions.DECREASE_LIQUIDITY,
+    Actions.TAKE_PAIR
+);
+```
+
+**2. Dust Handling**: Ignore small amounts
+
+```solidity
+bytes memory actions = abi.encodePacked(
+    Actions.DECREASE_LIQUIDITY,
+    Actions.CLEAR_OR_TAKE,  // For token0
+    Actions.CLEAR_OR_TAKE   // For token1
+);
+```
+
+### **Implementation**
+
+When removing liquidity from a position, you’ll be able to receive tokens and any accumulated fees. Let’s break down the implementation step by step.
+
+```solidity
+/// @notice Removes liquidity from a position
+/// @param tokenId The ID of the position
+/// @param liquidityDecrease Amount of liquidity to remove
+/// @param amount0Min Minimum amount of token0 to receive
+/// @param amount1Min Minimum amount of token1 to receive
+/// @param ignoreDust Whether to ignore small amounts
+/// @param recipient Address to receive the tokens
+function decreaseLiquidity(
+    uint256 tokenId,
+    uint128 liquidityDecrease,
+    uint256 amount0Min,
+    uint256 amount1Min,
+    bool ignoreDust,
+    address recipient
+) external {
+
+```
+
+Choose the appropriate delta resolution based on dust handling preference:
+
+```solidity
+// Define the sequence of operations:
+// If ignoring dust: Use CLEAR_OR_TAKE for each token
+// If not: Use TAKE_PAIR to receive all tokens
+bytes memory actions;
+if (ignoreDust) {
+    actions = abi.encodePacked(
+        Actions.DECREASE_LIQUIDITY, // Remove liquidity
+        Actions.CLEAR_OR_TAKE,      // Handle token0 with dust threshold
+        Actions.CLEAR_OR_TAKE       // Handle token1 with dust threshold
+    );
```

Comment:
yeah would probably remove this

forfeiting dust might happen when increasing liquidity and autocompounding fees

Reply:
removed